Documentation/operations/scalability/identity-relevant-labels.rst

```diff
@@ -90,20 +101,26 @@ will be used to evaluate Cilium identities:
 The above configuration would only include the following labels when evaluating
 Cilium identities:
 
-- io.kubernetes.pod.namespace*=.*
-- k8s-app*=*
-- app*=*
-- name*=*
+- k8s:k8s-app
+- k8s:app
+- k8s:name
+- reserved:.*
+- io.kubernetes.pod.namespace
+- io.cilium.k8s.namespace.labels
+- app.kubernetes.io
```

Comment:
These are not labels, these are 'keys', we should keep the "`=`" otherwise it might give the impression that a label is `k8s=k8s-app` for example.

Reply:
Would switching `s/the following labels/the following label keys/` work? The whole `*=*` was a bit ugly and hard to read IMO.